docs/user-guide/resource_tracking.md

```diff
@@ -1,8 +1,46 @@
 # Resource Tracking
 
+## Tracking Kubernetes resources by annotation
+
+Argo CD can be instructed to use the following methods for tracking:
+
+1. `annotation` (default) - Argo CD uses the `argocd.argoproj.io/tracking-id` annotation to track application resources. Use this when you don't need to maintain both the label and the annotation.
+1. `annotation+label` - Argo CD uses the `app.kubernetes.io/instance` label but only for informational purposes. The label is not used for tracking purposes, and the value is still truncated if longer than 63 characters. The annotation `argocd.argoproj.io/tracking-id` is used instead to track application resources. Use this for resources that you manage with Argo CD, but still need compatibility with other tools that require the instance label.
+1. `label` - Argo CD uses the `app.kubernetes.io/instance` label
+
+
+Here is an example of using the annotation method for tracking resources:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment
+  namespace: default
+  annotations:
+    argocd.argoproj.io/tracking-id: my-app:apps/Deployment:default/nginx-deployment
```

Comment:
extreme nit
```suggestion
    argocd.argoproj.io/tracking-id: my-app:apps/Deployment:default/my-deployment
```

Reply:
I live for reviews at this level of detail